### PR TITLE
Staking changes 1101

### PR DESCRIPTION
**Pull request by eshaben**
Update docs for new deferred staking payouts

### CODE REVIEW

File: node-operators/networks/collator.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Before getting started, it's important to note some of the timings of different
 
 
 !!! note 
-    Joining the collator candidate pool takes effect immediately. Adding or increasing a delegation also takes effect immediately, but rewards payouts begin {{ networks.moonriver.delegator_timings.rewards_payouts.rounds }} rounds later. The values presented in the previous table are subject to change in future releases.
+    Joining the collator candidate pool takes effect immediately. Adding or increasing a delegation also takes effect immediately, but rewards payouts are calculated {{ networks.moonriver.delegator_timings.rewards_payouts.rounds }} rounds later. The calculated rewards are then paid out incrementally, once per block, until all of the rewards have been paid. In each block, one collator is chosen to receive their reward payout, and is paid along with their delegators. The values presented in the previous table are subject to change in future releases.
```
Comment (themacexpert) [inline node-operators/networks/collator.md:86]: Can we include an example here such as, "Therefore, if there are 58 collators, the latest a collator candidate and their delegators would be paid their rewards is on the 58th block of the new round?"

File: tokens/staking/stake.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Token holders that want to easily stake their tokens can use the [Moonbeam dApp]
     | Maximum delegators per candidates |  |                                                    {{ networks.moonriver.staking.max_del_per_can }}                                                     |
```
Comment (themacexpert) [inline tokens/staking/stake.md:30]: Max delegators per candidate has increased from 100 to 500 on Moonriver and Moonbase Alpha

Comment (themacexpert) [reply tokens/staking/stake.md:30]: https://github.com/PureStake/moonbeam/pull/1096

**Comment by hyd628** [comment]
lgtm

**Comment by eshaben** [comment]
created PR #217 to incorporate Kevin's feedback! 🚀

**Pull request merged**